packages/truffle-core/lib/test.js

```diff
@@ -101,7 +102,7 @@ const Test = {
     await this.defineSolidityTests(
       mocha,
       testContracts,
-      dependencyPaths,
+      compilations.solc.sourceIndexes,
```

Comment:
Is this the same thing as `dependencyPaths` was??

Reply:
Yep

Reply:
Rad